pkg/assessor/manifest/manifest.go

```diff
@@ -210,6 +227,38 @@ func useADDstatement(cmdSlices map[int][]string) bool {
 	return false
 }
 
+func sensitiveVars(cmd string) bool {
+	if !strings.Contains(cmd, "=") {
+		return false
+	}
+
+	for _, s := range sensitiveWords {
+		sensitiveWords = append(sensitiveWords, strings.ToUpper(s))
```

Comment:
This line will duplicate the slice size every time it's executed. On an image with 10 env/run layers the slice size is going to grow to 16 * 2^10

If I understand this right you want to append uppercase words to match them as well, there's a better way. You're using regex so you can configure the regex match to be case insensitive or, even better, transform the input string to lower case instead.

There are also some edge cases you can't capture using this method, for example a variable named `Pass`.

Reply:
Removed this for, the regex is case insensitive now